app/controllers/events_controller.rb

```diff
@@ -0,0 +1,40 @@
+class EventsController < ApplicationController
+  def index
+    @events = Event.all
+  end
+
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = Event.new(event_params)
+    @event.creator = current_user
```

Comment:
Just a nitpick: you can merge these two lines into one, like this
```suggestion
    @event = current_user.created_events.build(event_params)
```

Reply:
uff! nice one